src/mcp/server/sse.py

```diff
@@ -156,11 +156,11 @@ async def handle_post_message(
             response = Response("Could not find session", status_code=404)
             return await response(scope, receive, send)
 
-        json = await request.json()
```

Comment:
The `json()` calls the `json` module, we can just use the body, and Pydantic's `model_validate_json`. It's faster.